.eslintrc.yaml

```diff
@@ -36,6 +36,7 @@ rules:
   unicorn/no-array-reduce: off
   unicorn/prevent-abbreviations: off
   unicorn/catch-error-name: off
+  no-param-reassign: off
```

Comment:
What's the reason for turning off the param reassign rule and conditional expect?

Reply:
I didn't want to deal with it just trying to get this built, but also param reassign is fairly fundamental to our teardown, since we make everything undefined, and the conditional expect rule is just annoying, it yells at you if you expect inside a promise, which is a really common pattern

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-conditional-expect.md

> The following patterns are warnings:
> ...
>  await foo().catch(error => expect(error).toBeInstanceOf(error));

Reply:
What's the reason to reassign the params to in teardown? Is that guarding for subsequent calls to methods on the `PlayerComponent`? Given we only do it for one I'd argue for just reassigning the param to a local variable in this case.

Reply:
It does have the "WIP" label which blocks merges

Reply:
Oh! Sorry assumed it was ready – didn't see the WIP label. Second time I review an unfinished PR cus I spot it in `announce-bsp`😅.

Reply:
It is kinda my fault for not making it a draft, DW, and you should definitely be questioning this if it was otherwise being shipped :D